### ISSUE TITLE
Problem with getting an image

### ISSUE THREAD

**Issue by DevYuns** [storybookjs/react-native#59]
**Describe the bug**
When I tried to use an image(png) with storybook, it didn't appear on the screen. Even though it doesn't return any error. But when I run expo start, It returns different result. I can see the image. but running from stroybook(yarn web), nothing happened.

please check this out.

**Code snippets**
```html
<Accordion
  key={i}
  isAnimated={true}
  contentVisible={false}
  duration={300}
  visibleElement={<Image source={IC_ARR_UP} />}
  ...
/>
```

**repository**

https://github.com/DevYuns/dooboo-ui/blob/378c49bfd896676a5540bd668dce76b449537f23/stories/dooboo-ui/Accordion.stories.tsx#L51-L63

**screenshots**

Run from `expo start --web`

<img width="375" alt="Screen Shot 2020-06-10 at 17 10 28" src="https://user-images.githubusercontent.com/58724686/84243802-af434e80-ab3d-11ea-8de3-fdc2894a881c.png">

Run from `start-storybook -p 6006`

<img width="369" alt="Screen Shot 2020-06-10 at 17 12 18" src="https://user-images.githubusercontent.com/58724686/84243778-a9e60400-ab3d-11ea-870f-0a1bae270779.png">

**System:**
System:
    OS: macOS 10.15.5
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  Binaries:
    Node: 14.2.0 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.4 - /usr/local/bin/npm
  Browsers:
    Chrome: 83.0.4103.97
    Safari: 13.1.1
  npmPackages:
    @storybook/addon-actions: ^5.3.19 => 5.3.19 
    @storybook/addon-info: ^5.3.19 => 5.3.19 
    @storybook/addon-knobs: ^5.3.19 => 5.3.19 
    @storybook/addon-links: ^5.3.19 => 5.3.19 
    @storybook/addon-notes: ^5.3.19 => 5.3.19 
    @storybook/addons: ^5.3.19 => 5.3.19 
    @storybook/preset-typescript: ^3.0.0 => 3.0.0 
    @storybook/react: ^5.3.19 => 5.3.19 
    @storybook/react-native: ^5.3.19 => 5.3.19

**Comment by dannyhw** [comment]
I've never seen any problems loading images before, could you please share more about this. For example what is the format of the image? If it's an svg you need to use an [svg library](https://github.com/react-native-community/react-native-svg) to make it work.

**Comment by DevYuns** [comment]
> I've never seen any problems loading images before, could you please share more about this. For example what is the format of the image? If it's an svg you need to use an [svg library](https://github.com/react-native-community/react-native-svg) to make it work.

I have used `png` in `styled.image` with `styled-component`. `Svg` is ok, but `png` doesn't appeared in storybook.

**Comment by dannyhw** [comment]
sorry I haven't been able to investigate this yet. Still having this issue? Have you tried the regular image component without using styled? Can you check if image appears as expected outside of storybook? There shouldn't be any reason that storybook would effect your image.

**Comment by dannyhw** [comment]
this should be solved in 6.5+
